### ISSUE TITLE
[02/02] today I read

### ISSUE THREAD

**Issue by nmhkahn** [nmhkahn/paper-notes#1]
## NeRF: Representing Scenes as Neural Radiance Fields for View Synthesis 
- 기존 novel view synthesis 방법들과 달리 매우 간단한 MLP만 사용
- MLP는 (x, y, z) + (theta, phi)를 입력으로 받고 (color, density)를 출력하고, differentiable render를 사용해서 학습
- MLP는 low-frequency를 주로 학습하는 경향이 있어서 별도의 high-frequency를 학습하도록 PE를 추가로 줌
    - Transformer와 달리 PE는 ordering보다는 입력 coord를 high-dimension space로 보내는 역할을 한다고 하는듯

## Adversarial generation of continuous images
- NeRF처럼 이미지를 p = (x, y) -> v = (r, g, b)로 맵핑하는 연속함수 F(p)로 보고, 이 함수 F를 MLP로 approximation 하는 문제로 품 (implicit neural representation; INR)
- noize z ~ N(.)를 이용해서 synthesis network의 파라미터 theta를 구하는 hyperparameter 방식의 기법을 사용함
- 나이브하게 하면 연산량이 많으니까 factorized multiplicative modulation / multi-scale 기법 사용
- Discussion section에서 INR의 potential을 얘기하는게 이게 재밌음:
    - PE를 사용하기 때문에 입력까지 backprop이 가능해서 spatial transformer와 같은 방법들을 적용하기 편해짐
    - 1x1 conv를 사용하는것과 동일해서 파라미터 prediction만 적절하게 된다면 기존 CNN기반 decoder보다 빠름 (대신 파라미터는 증가하는데 inference speed가 더 중요하니까)
    - Local context에 depend하지 않기 때문에 autogressive나 CNN과 달리 각 픽셀을 병렬적으로 처리할 수 있음
    - Universal decoder architecture: 어떤 이미지 해상도 / 도메인이 들어오더라도 기본적인 MLP 구조는 변경하지 않고, 각 task에 맞는 PE와 파라미터 prediction network만 변경하면 됨
    - Biological plausibility: conv는 receptive field 개념이 biological inspired 된게 맞는데 conv를 여러층 쌓은 decoder도 그렇게 말할 수 있을까? 오히려 hypernetwork가 생물학적으로 밀접한 관련이 있다는 연구가 있음
    - sidenote: conv 자체가 어쨌거나 강한 inductive bias (receptive field)를 주는건 맞아서 (+ 생물학적인 bias) 이거랑 같이 쓸순 없을까? 아직 뭔가 receptive field를 안보고 하는게 되게 어색함

## Image generators with conditionally-independent pixel synthesis
- 위 adversarial generation of continuous images와 방법이 동일함
- 입력으로는 pe기반 feature인 fourier feature + coord-embed (자세한 설명이 없는데 대충 coord를 word로 보고 embed한듯)
 
## Spatially-Adaptive Pixelwise Networks for Fast Image Translation
- MLP 기반으로 image-to-image translation
- INR처럼 하는대신, z->theta가 아니라 low-res input에서 MLP 파라미터를 predict하고 이를 nearest 한 방식으로 upsample 한뒤에 원래 해상도에서 MLP
- 기존 CC-FPSE, SPADE같은 SOTA보다 빠르면서 comparable한 성능을 보임

## Learning Semantic-aware Normalization for Generative Adversarial Networks
- 코드가 없어서 논문만으로 내부구조를 100% 이해하기 어려움;
- stylegan을 응용했는데, z를  inter-, intra-group로 나누고 (각각 z, h로 denote), z는 group내에서 norm (grouped AdaIN) h는 그룹 밖에서 relative importance를 위해 AdaGN에서 사용
- 각 channel이 group에 매핑되도록 group loss같은걸 넣고, disentanglement를 높이기 위해 mutual loss를 도입함
- sidenote: modulation을 group단위로 둬서 여러 task에 적합하도록?

---
업데이트 예정

## Invertible Image Rescaling
